### PR TITLE
Header improvements

### LINKED ISSUE #3: page header

### PR DESCRIPTION
**Pull request by kulla**
Fixes #3

### CODE REVIEW

**Comment by kulla** [comment]
@Entkenntnis I'll plan to continue my work on Sunday...

**Comment by elbotho** [comment]
Okay! Pretty happy with it now.

File: src/components/navigation/SearchInput.tsx [hunk 1]
```diff
@@ -153,20 +157,29 @@ const _Input = styled.input<{ focused: boolean }>`
...
+  ::-webkit-input-placeholder {
+    opacity: 1;
+  }
+  ::-moz-placeholder {
+    opacity: 1;
+  }
+  :-ms-input-placeholder {
+    opacity: 1;
+  }
+
```
Comment (Entkenntnis) [inline src/components/navigation/SearchInput.tsx]: you can add `opacity: 1` to the rule in 162-164 and remove the vendor-prefixes. I just checked it, styled components are generating them automatically

File: src/components/navigation/Menu.tsx [hunk 2]
```diff
@@ -24,91 +29,118 @@ export default function Menu(props) {
...
   display: inline-block;
 `
 
+const linkStyle = css`
+  &:active,
+  &:hover,
+  &[aria-expanded='true'] {
+    color: #fff;
+    background-color: ${props => props.theme.colors.brand};
+  }
```
Comment (Entkenntnis) [inline src/components/navigation/Menu.tsx]: nice

**Comment by Entkenntnis** [comment]
running prettier on csshelper will solve the conflict

File: src/components/navigation/SearchInput.tsx (hunk 1 above)
Comment (elbotho) [reply src/components/navigation/SearchInput.tsx]: nice!

**Comment by elbotho** [comment]
@Entkenntnis thank you for the feedback!
Should be good to go now.

**Pull request merged**